src/content/en/updates/2019/03/prefers-reduced-motion.md

```diff
@@ -0,0 +1,231 @@
+[🐈 HTTP status cats](https://http.cat/). First, take a moment to appreciate the joke, it's
+hilarious and I'll wait. Now that you're back, let me introduce the
+[demo](https://prefers-reduced-motion.glitch.me). When you scroll down, each HTTP status cat
+alternatingly appears from either the right or the left side. It's a buttery smooth 60fps animation,
+but as outlined above, some users may dislike it or event get motion sick by it, so the demo is
+programmed to respect `prefers-reduced-motion`. This even works dynamically, so users can change
+their preference on-the-fly, no reload required. If a user prefers reduced motion, the non-necessary
+reveal animations are gone, and just the regular scrolling motion is left. The screencast below
+shows the demo in action:
+
+<video width="1098" height="870" intrinsicsize="1098x870" controls>
+  <source src="../../videos/2019/03/prefers-reduced-motion/prefers-reduced-motion.mp4" type="video/mp4">
+</video>
+
+## (Bonus) Forcing reduced motion on all websites
+
+Not every site will use `prefers-reduced-motion`, or maybe not consequently enough for your taste.
+If you, for whatever reason, want to stop motion on all websites, you actually can. One way to make
+this happen is to inject a stylesheet with the following CSS into every webpage you visit.
+There are several
+[browser extensions](https://chrome.google.com/webstore/search/user%20stylesheets?_category=extensions)
+out there (use at your own risk!) that allow for this.
+
+```CSS
+@media (prefers-reduced-motion: reduce) {
+  * {
+    animation-duration: 0.001s !important;
+    transition-duration: 0.001s !important;
+  }
+}
+```
+
+The way this works is that the CSS above overrides the durations of all animations and transitions
+to such a short time that they are not noticeable anymore. As some websites depend on an animation
+to be run in order to work correctly (maybe because a certain step depends on the firing of the
+[`animationend` event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/animationend_event)),
+the more radical `animation: none !important;` approach wouldn't work. Even the above hack is not
+guaranteed to succeed on all websites, so be sure to deactivate it when you notice breakage.
+
+## Conclusions
+
+Respecting user preferences is key for modern websites, and browsers expose more and more features
+to enable web developers to do so. The CSS Working Group are currently standardizing more
+[user preference media queries](https://drafts.csswg.org/mediaqueries-5/#mf-user-preferences)
+like `prefers-reduced-transparency` (detects if the user prefers reduced transparency),
+`prefers-contrast` (detects if the user has requested the system increase or decrease the amount of
+contrast between adjacent colors), `prefers-color-scheme` (detects if the user prefers a light or
+dark color scheme), and `prefers-reduced-transparency` (detects if the user prefers reduced
```

Comment:
I think you have prefers-reduced-transparency in this list twice.

Reply:
Ooops, good catch. Also added links to all media queries.